### ISSUE TITLE
Customized training function with epochs input

### ISSUE THREAD

**Issue by Derek-Wds** [sql-machine-learning/sqlflow#2755]
**Description**
I'm trying to use the `sqlflow_train_loop` api to build a customized training scripts, however, I found this function does not accept the `epochs` as the function argument [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/python/runtime/tensorflow/train_keras.py#L130). I understand for the deep clustering model [example](https://github.com/sql-machine-learning/models/blob/develop/sqlflow_models/deep_embedding_cluster.py#L232), the `epochs` argument is not used. However, I'm not sure if other customized models would be able to use `epochs` as on one of the input arguments in the `sqlflow_train_loop` function.

I wonder if this is a bug for the `keras_train_compiled` function, or there exists other ways to do it. Thanks!

**Comment by typhoonzero** [comment]
In fact, you can define the model `__init__` function like `__init__(self, feature_columns, epochs=1, batch_size=32)` then write statement like `SELECT * FROM train_table TO TRAIN GCNClassifier WITH model.epochs=10 LABEL class INTO ...;`

**Comment by Derek-Wds** [comment]
Yeah, I see. I was thinking not include training parameters as part of the model's attributes, but now it seems to be necessary to do so. Thanks for helping out!
